client/src/config/Routes.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
moved `Routes.jsx` into the `/config` directory alongside `Theme.js` - what are your thoughts?